components/url/lib.rs

```diff
@@ -49,8 +49,8 @@ impl ServoUrl {
 
     // NOTE: These methods return options that are always true temporarily until
     // we special-case some urls to avoid going through rust-url.
```

Comment:
This comment should go away if we're making this change. I don't understand it though; any comments, @emilio?